docs/source/quick-start.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
What changed in the notebook?

Reply:
See you give plenty of more detail. Great

Reply:
I didn't change anything. Not sure what happened here. I should re-run the notebook

Reply:
Maybe just download the current master notebook then